eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -483,6 +489,9 @@ jobs:
           find artifacts/log/binary-report/ -exec rsync -R {} -t ${targetFolder} \;
         fi
 
+        # Needed to diagnose https://github.com/dotnet/dnceng/issues/3305. Remove once the issue is resolved.
+        find src/ -type f -name "*.dmp" -exec rsync -R {} -t ${targetFolder} \;
```

Comment:
Removing. This isn't applicable in `src/` because the dmps are being produced in vmr/artifacts.